### PR TITLE
chore: make markdown style scope

### PR DESCRIPTION
**Pull request by boojack**
related PR: #280

### CODE REVIEW

File: layouts/default.vue [hunk 1]
```diff
@@ -51,11 +51,14 @@ export default defineComponent({
       }
     });
 
-    watch(route, () => {
-      contentElementRef.value?.scrollTo({
-        top: 0,
-      });
-    });
+    watch(
+      () => route.value.path,
```
Comment (Megrax) [inline layouts/default.vue:55]: What's the difference between using `route` and `() => route.value.path` ?

Comment (boojack) [reply layouts/default.vue:55]: diff like this:

image

We only need to detect the path changed to scroll container to page top with `nuxt-link`.

**Review by Megrax — APPROVED** [review]
LGTM

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**